packages/main/src/CheckBox.ts

```diff
@@ -405,13 +418,27 @@ class CheckBox extends UI5Element implements IFormElement {
 
 	get effectiveTabIndex() {
 		const tabindex = this.getAttribute("tabindex");
-		return this.disabled ? undefined : tabindex || "0";
+		return this.disabled || this.displayOnly ? undefined : tabindex || "0";
 	}
 
 	get isCompletelyChecked() {
 		return this.checked && !this.indeterminate;
 	}
 
+	get isDisplayOnly() {
+		return this.displayOnly && !this.disabled;
+	}
+
+	get displayOnlyIcon() {
+		if (this.isCompletelyChecked) {
```

Comment:
I think we only the import "@ui5/webcomponents-icons/dist/accept.js"; icon.
Do we need more icons imports - complete, tri-state, border

Reply:
Yes, I forgot to add them. Thanks!